backend/recipes/serializers.py

```diff
@@ -37,20 +50,17 @@ class Meta:
             "ingredients",
         ]
```

Comment:
I haven't checked the Recipe model in a long time so I completely forgot what it looks like (:joy:) but we might be able to use `fields = "__all__"` here too if we need everything from the Recipe.